src/main/kotlin/domain/CarRacingGame.kt

```diff
@@ -0,0 +1,33 @@
+package domain
+
+import data.Car
+import data.CarPath
+import data.CarRacingGamePlayer
+
+class CarRacingGame {
+
+    fun startDriving(players: List<CarRacingGamePlayer>): List<CarPath> {
+        val carsPath = mutableListOf<CarPath>()
+
+        players.forEach { player ->
+            carsPath.add(player.moveCar())
+        }
+
+        return carsPath.toList()
```

Comment:
이런 형태의 로직은 코틀린 확장함수를 활용하면 깔끔히 쓸 수 있답니다 :)
```suggestion
        return players.map { it.moveCar() }
```